0001-consolidate-terraform.md

```diff
@@ -0,0 +1,52 @@
+# Consolidate Terraform
+
+* Status: proposed
+* Deciders: @ZavenArra, Peter Willis, Tuan-Ahn Nguyen, MattK
+* Date: 2022-03-01 
+
+Technical Story: Where shoud our terraform code be located? 
+
+## Context and Problem Statement
+
+Current terraform is spread through the treetracker-infrastructure repository.  We want to consolidate this IaC in a way that makes it easy to to find and easy for new volunteers to contribute to.  We don't want other kinds of infra codes, such as scripts or deployment jobs, to create extra mental overhead. 
+
+
+## Decision Drivers <!-- optional -->
+
+* Clean organizational strategy
+* Ease in which new volunteers (some rather junior) can find terraform codes
+* Ease in which new volunteers can contribute or create new modules
+
+## Considered Options
+
+* Create a repository just for terraform and related utility scripts
+* Organize treetracker-infrastructure in subfolders: terraform, application platform, utilitis
```

Comment:
With not enough depth in infrastructure code, I am not sure how relevant my observation here is. But I like the option 2, terraform is neatly placed within infrastructure.